### ISSUE TITLE
should we just use the church theme?

### ISSUE THREAD

**Issue by lmkirvan** [kirvanai/Gainesville-Quakers-Failed#1]
https://themes.gohugo.io/themes/alpha-church/

**Comment by kirvanai** [comment]
Yeah that one looks pretty good. Do you think you’ll want an events page or
something like that?

**Comment by lmkirvan** [comment]
Yeah I think the pages are calendar/ new to quakerism/ blog (maybe called newsletter?) / Main page (with map embedded)

**Comment by lmkirvan** [comment]
And the calendar is probably just an embedded Google calendar. Not sure how that works? Maybe just an iframe or maybe some kind of web hook?
